backend/apis/flag-utils/is-class-b-misdemeanor.js

```diff
@@ -0,0 +1,15 @@
+exports.isClassBMisdemeanor = function(parsedObj) {
+  // Determine if the individual has at least 4 convictions where 3 are Misdemeanor B's
+  // must have at least 4 convictions
+  if (parsedObj.charges.length < 4) {
+    return false;
+  }
+  let numOfClassBMisdemeanors;
+  // 3 or more of the convictions must be Class B Misdemeanors
+  for (let i = 0; i++; i < parsedObj.charges.length) {
+    if (parsedObj.charges[i].includes("B Misdemeanor")) {
+      numOfClassBMisdemeanors++;
+    }
```

Comment:
💡 i think you could use `.reduce` here and it would be a little cleaner/more concise.